dom-creation/main.js

```diff
@@ -0,0 +1,91 @@
+/* exported pokedex */
+function renderPokemon(object) {
+  var columnDiv = document.createElement('div');
+  columnDiv.setAttribute('class', 'column-third');
+  var pokeCard = document.createElement('div');
+  pokeCard.setAttribute('class', 'pokemon-card');
+  columnDiv.appendChild(pokeCard);
+  var cardTextDiv = document.createElement('div');
+  cardTextDiv.setAttribute('class', 'pokemon-card-text');
+  pokeCard.appendChild(cardTextDiv);
+  for (const key in object) {
+    if (key === 'imageUrl') {
+      var image = document.createElement('img');
+      image.setAttribute('src', `${object[key]}`);
+      pokeCard.insertBefore(image, cardTextDiv);
+    } else if (key === 'description') {
+      var description = document.createElement('p');
+      description.textContent = `${object[key]}`;
+      cardTextDiv.appendChild(description);
+    } else if (key === 'number') {
+      var number = document.createElement('h3');
+      number.textContent = `${object[key]}`;
+      cardTextDiv.insertBefore(number, cardTextDiv.lastChild);
+    } else if (key === 'name') {
+      var name = document.createElement('h2');
+      name.textContent = `${object[key]}`;
+      cardTextDiv.insertBefore(name, cardTextDiv.firstChild);
+    }
+  }
```

Comment:
Since you already know the pokemon object property names you're looking for, it'd be far easier to pull those values directly rather than `for ... in` loop over the pokemon object's property names.